client/lib/interval/index.js

```diff
@@ -0,0 +1,90 @@
+import React, { PropTypes } from 'react';
+import omit from 'lodash/object/omit';
+
+import { add, remove } from './runner';
+
+export const EVERY_SECOND = 1000;
+export const EVERY_FIVE_SECONDS = 5 * 1000;
+export const EVERY_TEN_SECONDS = 10 * 1000;
+export const EVERY_THIRTY_SECONDS = 30 * 1000;
+export const EVERY_MINUTE = 60 * 1000;
+
+/**
+ * Calls a given function on a given interval
+ */
+export default React.createClass( {
+	displayName: 'Interval',
+
+	propTypes: {
+		onTick: PropTypes.func.isRequired,
+		period: PropTypes.oneOf( [
+			EVERY_SECOND,
+			EVERY_FIVE_SECONDS,
+			EVERY_TEN_SECONDS,
+			EVERY_THIRTY_SECONDS,
+			EVERY_MINUTE
+		] ).isRequired,
+		pauseWhenHidden: PropTypes.bool,
+		children: PropTypes.element
+	},
+
+	getDefaultProps: () => ( {
+		pauseWhenHidden: true,
+		children: <span />
```

Comment:
I feel a bit of shame about this, but I believe I did that because the tests were failing with a `null` return. I have updated the code and the tests in e34d0a7 to handle this properly with a `null`